src/Config.php

```diff
@@ -2,31 +2,15 @@
 
 namespace PHPOnCouch;
 
-
-use Dotenv\Dotenv;
-use Dotenv\Exception\InvalidPathException;
-
 class Config
 {
 
     private static $instance = null;
     private static $adapterKey = 'HTTP_ADAPTER';
     private $config;
 
-    private function __construct()
+    private function __construct(string $adapter = 'curl')
     {
-        $env = Dotenv::createImmutable(__DIR__);
-        try {
-            $env->load();
-        } catch (InvalidPathException $e) {
-            //The file is not available :(
-        }
-        //Default
-        if (empty(getenv(self::$adapterKey)))
-            putenv(self::$adapterKey . "=curl");
-
-        $env->required(self::$adapterKey)->allowedValues(['curl', 'socket']);
```

Comment:
Regression here. We still need to be able to define the 'HTTP_ADAPTER' through environment variables if required. 

The values can either be 'curl' or 'socket'.

If it's another value, it should fail.
If no value is specified, we fallback to `curl`